### PR TITLE
Changed the supported enclave versions

### PR DESCRIPTION
**Pull request by enclave-tom**
I've also removed the `purge` command as currently it's use doesn't match our description this is being looked at as part of https://github.com/enclave-networks/fabric/issues/782

### CODE REVIEW

**Comment by enclave-marc-barry** [comment]
Why have we dropped support for 14.04, Debian 9 and Fedora 32?

**Comment by enclave-marc-barry** [comment]
Ok, I see - linking this back to https://github.com/enclave-networks/fabric/issues/825

File: docs/setup/linux.md [hunk 1]
```diff
@@ -265,10 +265,6 @@ You can configure your linux distribution to automatically update Enclave to ens
 
         sudo apt remove enclave
 
-    Remove the Enclave package and any configuration files (including private any keys generated during enrolment).
```
Comment (enclave-alistair) [inline docs/setup/linux.md:268]: Should we add a separate section after the distro tabs telling people how to delete their profiles?

**Review by enclave-alistair — CHANGES_REQUESTED** [review]

File: docs/setup/linux.md (hunk 1 above)
Comment (enclave-tom) [reply docs/setup/linux.md:268]: can do it should be as simple as `sudo rm -rd /etc/enclave/`

**Review by enclave-alistair — CHANGES_REQUESTED** [review]

**Review by enclave-alistair — APPROVED** [review]

**Pull request merged**